src/nsrt_learning/side_predicate_learning.py

```diff
@@ -354,6 +354,10 @@ def _initialize_general_pnad_for_option(
         preconditions = induce_pnad_preconditions(pnad)
         pnad.op = pnad.op.copy_with(preconditions=preconditions)
 
+        # Finally, remove the side predicates from these operators.
+        # These will be filled in later.
+        pnad.op = pnad.op.copy_with(side_predicates=set())
```

Comment:
are we doing this so we can remove the "add_effects" semantics stuff?

Reply:
yes - that was the intent